### PR TITLE
Create Get-GitModules.ps1

### PR DESCRIPTION
**Pull request by ToniaSanzo**
Add the **Invoke-Download.ps1** script to the **SDL_image/external** directory. The **Invoke-Download.ps1** script is a Windows OS port of the **download.sh** script in **SDL_image/external**. The script clones the Github repositories specified in the **SDL_image/.gitmodules** file.

### CODE REVIEW

File: external/Invoke-Download.ps1 [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<#
+  .SYNOPSIS
+  Downloads the Git modules specified in ../.gitmodules
+
+  .DESCRIPTION
+  Parses and downloads the Github repositories specified in the .gitmodules file
+
+  .EXAMPLE
+  PS> .\Get-GitModules.ps1
```
Comment (madebr) [inline external/Invoke-Download.ps1:9]: I think `external/Get-GitModules.ps` would be a nicer name for this powershell script.

**Review by madebr — APPROVED** [review]
With a file rename, lgtm.
Btw, this is not Windows-only. PowerShell also works on other operating systems: https://github.com/PowerShell/PowerShell/releases/tag/v7.3.9

File: external/Invoke-Download.ps1 (hunk 1 above)
Comment (ToniaSanzo) [reply external/Invoke-Download.ps1:9]: will do

Comment (ToniaSanzo) [reply external/Invoke-Download.ps1]: done

File: external/Get-GitModule.ps1 [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<#
+  .SYNOPSIS
+  Downloads the Git modules specified in ../.gitmodules
+
+  .DESCRIPTION
+  Parses and downloads the Github repositories specified in the .gitmodules file
+
+  .EXAMPLE
+  PS> .\Get-GitModules.ps1
```
Comment (madebr) [inline external/Get-GitModule.ps1:9]: Little typo in the file name. It should be plural, so "Get-GitModule**s**.ps1".

Comment (ToniaSanzo) [reply external/Get-GitModule.ps1]: Good catch, thank you, it should be Get-GitModules.ps1 now.

**Comment by madebr** [comment]
Thank you for the pr!
I think lots of Windows devs will thank you!

**Comment by slouken** [comment]
Thank you! Can we copy this to the other SDL satellite libraries as well?

**Comment by ToniaSanzo** [comment]
Absolutely

**Comment by madebr** [comment]
The script is now available in the SDL_image, SDL_ttf and SDL_mixer repo's for both SDL2 and SDL3.

**Comment by ToniaSanzo** [comment]
my pleasure 🙂

**Pull request merged**